client/js/components/month/month-week-board.js

```diff
@@ -0,0 +1,20 @@
+/** @jsx React.DOM */
+var React = require('react');
+var MonthEvent = require('./month-event.js');
+var _ = require('underscore');
+
+var MonthWeek = React.createClass({
+  render: function() {
+    var weekEvents = _.map(this.props.week, function(day, key) {
+      return <MonthEvent key = { key } day = { day } />
```

Comment:
Not sure the name MonthEvent makes sense considering we are not talking about a single event here. Rather, we are talking about the data for a single day that discusses the number events and the health of the entire day full of events.